rar.go

```diff
@@ -10,8 +10,12 @@ import (
 	"github.com/nwaples/rardecode"
 )
 
+// Rar is for RAR archive format
+var Rar Archiver
```

Comment:
Can you just initialize this here, instead of in init? `var Are = RarFormat` for this one, and similarly for the other formats.